packages/zoe/src/contracts/coveredCall-durable.js

```diff
@@ -72,24 +67,16 @@ const sellSeatExpiredMsg = `The covered call option is expired.`;
  * specified in the invitation value, and want the underlying assets
  * exactly.
  *
- * @param {import('@agoric/vat-data').Baggage} installationBaggage
+ * @param _privateArgs unknown
+ * @param {import('@agoric/vat-data').Baggage} instanceBaggage
  * @param {ZCF} zcf passed in setupInstallation, but not completely initialized
```

Comment:
I don't think _privateArgs declaration syntax is correct.
@param declarations must be in the same order as the params

```suggestion
 * @param {ZCF} zcf passed in setupInstallation, but not completely initialized
 * @param {unknown} _privateArgs
 * @param {import('@agoric/vat-data').Baggage} instanceBaggage
```

Reply:
Fixed by #5772